### PR TITLE
`frame_width` and `frame_height` settings

### PR DESCRIPTION
**Pull request by yvon**
@spajus @EddieOne

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -94,6 +94,24 @@ def after_create_account(user, auth)
               authenticator: OAuth2BasicAuthenticator.new('oauth2_basic'),
               message: "OAuth2"
 
+after_initialize do
+  auth_providers.each do |auth|
+    # SiteSetting unavailable before Framework initialize (constant undefined,
+    # ActiveRecord connection not configured)
+    width = SiteSetting.frame_width
```
Comment (spajus) [inline plugin.rb:101]: shouln't this be `SiteSettings.login.oauth2_frame_width`?

Comment (yvon) [inline plugin.rb:101]: thanks

Comment (yvon) [inline plugin.rb:101]: Actually `SiteSettings.oauth2_frame_width`

**Comment by yvon** [comment]
Invalid PR.

```
couldn't find file '/home/yvon/discourse/plugins/discourse-oauth2-basic/auto_generated/plugin_50f64cdcd9a52eba11d2be31d6506c7def11eadd.js'
  (in /home/yvon/discourse/app/assets/javascripts/application.js.erb)
```

**Pull request closed without merging**